cli/crates/federated-dev/src/dev/websockets/messages.rs

```diff
@@ -0,0 +1,44 @@
+//! Message definitions for the [GraphQLOverWebsocket protocol][1]
```

Comment:
Could be in the gateway-v2 crate later as this would be shared between api & cli. Supposing we support it in api^^